2025-04-25 00:12:12,126 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:14:24,420 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:14:24,741 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:20:45,035 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:109]
2025-04-25 00:22:20,417 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:110]
2025-04-25 00:22:53,534 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:23:55,187 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:28,264 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:28,582 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 00:24:32,643 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-04-25 00:40:32,720 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-04-25 11:43:02,235 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-25 11:43:02,669 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:03:35,442 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:03:35,661 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:05:01,421 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:05:01,638 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:05:07,057 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:146]
2025-04-26 00:12:46,417 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 00:12:46,634 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:113]
2025-04-26 01:32:33,514 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 01:32:33,519 ERROR: Ошибка при инициализации модуля артикулов: Cannot create a consistent method resolution order (MRO) for bases Model, BaseModel, TimestampMixin [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 01:32:33,749 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 01:32:33,752 ERROR: Ошибка при инициализации модуля артикулов: Cannot create a consistent method resolution order (MRO) for bases Model, BaseModel, TimestampMixin [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 01:33:42,996 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 01:33:42,999 ERROR: Ошибка при инициализации модуля артикулов: Cannot create a consistent method resolution order (MRO) for bases Model, BaseModel, TimestampMixin [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 02:02:12,316 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 02:02:12,325 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 02:02:12,555 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 02:02:12,561 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 02:04:46,224 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 02:04:46,231 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 02:05:19,111 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 02:05:19,118 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 02:05:35,724 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 02:05:35,730 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:05:51,533 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:05:51,539 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:05:51,921 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:05:51,928 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:47:05,106 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:47:05,107 ERROR: Ошибка при инициализации модуля артикулов: cannot import name 'init_app' from 'app.modules.articles' (/Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py) [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 23:47:14,775 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:47:14,782 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-26 23:47:14,782 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:48:01,167 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:48:01,169 ERROR: Ошибка при инициализации модуля артикулов: cannot import name 'Article' from 'app.modules.articles.models' (/Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/models.py) [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 23:48:11,848 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:48:11,852 ERROR: Ошибка при инициализации модуля артикулов: cannot import name 'ArticleAttribute' from 'app.modules.articles.models' (/Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/models.py) [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:138]
2025-04-26 23:49:01,397 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:49:01,399 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-26 23:49:01,399 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:49:15,133 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:49:15,136 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-26 23:49:15,136 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-26 23:56:18,923 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-26 23:56:18,926 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-26 23:56:18,926 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:00:14,091 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:00:14,095 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:00:14,095 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:00:23,763 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:00:23,766 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:00:23,766 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:12:19,169 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:12:19,172 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:12:19,172 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:12:19,545 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:12:19,548 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:12:19,548 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:13:07,237 WARNING: 404 ошибка: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:156]
2025-04-27 00:23:54,034 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:23:54,037 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:23:54,037 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:23:54,266 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:23:54,269 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:23:54,269 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:23:54,517 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:23:54,520 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:23:54,520 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:24:20,496 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:24:20,499 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:24:20,499 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:24:20,721 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:24:20,724 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:24:20,724 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:37:00,054 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:37:00,057 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:37:00,057 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
2025-04-27 00:37:00,438 INFO: ERP_VITTAVENTO запущена [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:116]
2025-04-27 00:37:00,441 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/modules/articles/__init__.py:34]
2025-04-27 00:37:00,441 INFO: Модуль артикулов успешно инициализирован [in /Users/alekseyglumov/Projects/ERP_VITTAVENTO/app/__init__.py:136]
